recipe/meta.yaml

```diff
@@ -24,9 +24,10 @@ requirements:
   host:
     - python
     - pip
-    - cython >=0.28.3
+    - cython >=0.29.29
     - numpy
     - libgdal
```

Comment:
Missing `setuptools`, see https://github.com/Toblerity/Fiona/blob/bae44db5845955870a850fb53f67c9a4ab4c55d4/pyproject.toml#L5

Reply:
`certifi  # [py<=37]` is required in `host` and `run` https://github.com/conda-forge/fiona-feedstock/commit/d9e4aa7c746bbf88de69db5b269ce85e503c5541